docs/design/StratisSoftwareDesign.lyx

```diff
@@ -7210,12 +7173,26 @@ Encrypted device creation
 
 \begin_layout Standard
 Stratis does not wipe the device prior to initializing it with a LUKS2 header.
- Due to how data is encrypted with LUKS2, if a user is concerned with an
- attacker being able to determine exactly how much of the data on the device
- is encrypted, the device should be wiped with random data prior to providing
- it to a Stratis pool.
+ This means that an attacker can potentially deduce the following from examing
+ a disk that has been initialized using Stratis without any prior preparation
+ outside of Stratis:
+\end_layout
+
+\begin_layout Itemize
+The amount of data currently stored on disk that is encrypted.
+\end_layout
+
+\begin_layout Itemize
+The contents of unencrypted data that was previously stored on the disk
+ prior to initialization by Stratis.
+\end_layout
+
+\begin_layout Standard
+If a user is concerned with an attacker being able to determine exactly
```

Comment:
It might be better to split out the two remediations you describe below into the descriptions of the problem above.

So for, ```The contents...``` lines just follow it up with another sentence, like: To prevent an attacker from reading previously written unencrypted data, the disk should be zeroed or overwritten with random data before being presented to Stratis.